src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -2278,16 +2283,15 @@ private bool RefMustBeObeyed(bool isDelegateCreation, ArgumentSyntax argumentSyn
                 {
                     case BoundKind.PropertyAccess:
                     case BoundKind.IndexerAccess:
-                        return CheckIsVariable(argumentSyntax, arg, BindValueKind.RefOrOut, false, diagnostics);
+                        CheckIsVariable(argumentSyntax, arg, BindValueKind.RefOrOut, false, diagnostics);
+                        return;
```

Comment:
Yes, thanks.

---
In reply to: [122234878](https://github.com/dotnet/roslyn/pull/20121#discussion_r122234878) [](ancestors = 122234878)